benchmark/src/main/scala/mesosphere/marathon/core/health/impl/AppHealthCheckBenchmark.scala

```diff
@@ -0,0 +1,105 @@
+package mesosphere.marathon
+package core.health.impl
+
+import java.util.concurrent.TimeUnit
+
+import mesosphere.marathon.core.health.{Health, MarathonHttpHealthCheck, PortReference, impl}
+import mesosphere.marathon.core.health.impl.AppHealthCheckActor.{ApplicationKey, InstanceKey}
+import mesosphere.marathon.core.instance.Instance
+import mesosphere.marathon.state.PathId
+import org.openjdk.jmh.annotations._
```

Comment:
In general we try to avoid `_` imports in the recent past. Could you please address here and in the other places of this patch?

Reply:
I think it's fine here. The other benchmarks (that we wrote) do the same.

Reply:
I've removed _ imports in files where I've introduced it.